src/vs/workbench/browser/dnd.ts

```diff
@@ -172,14 +174,25 @@ export class ResourcesDropHandler {
 	}
 
 	async handleDrop(event: DragEvent, resolveTargetGroup: () => IEditorGroup | undefined, afterDrop: (targetGroup: IEditorGroup | undefined) => void, targetIndex?: number): Promise<void> {
-		const untitledOrFileResources = extractResources(event).filter(r => this.fileService.canHandleResource(r.resource) || r.resource.scheme === Schemas.untitled);
+
+		// {{SQL CARBON EDIT}}
```

Comment:
This logic should go in `src/vs/workbench/browser/parts/editor/editorDropTarget.ts` instead of `src/vs/workbench/browser/dnd.ts` since we only want to change the behavior of drag and drop in the editor. Put this logic in the `handleDrop` function of `src/vs/workbench/browser/parts/editor/editorDropTarget.ts`